web/source/kmwuifloat.js

```diff
@@ -487,13 +487,6 @@ try {
       return true;
     });               
 
-  /**
-   * Function     oncontrolblurred
-   * Scope        Public
-   * @param       {Object}  params  event object
-   * @return      {boolean}   
-   * Description  UI code to be executed on losing focus
-   */
   keymanweb['addEventListener']('controlblurred',
```

Comment:
This code is adding an event handler, not a public method.  Closure didn't like that it was being annotated as if our anonymous inline function were a public, named function.